multipaz-zkp/src/commonMain/kotlin/org/multipaz/zkp/mdoc/Attribute.kt

```diff
@@ -0,0 +1,7 @@
+package org.multipaz.zkp.mdoc
+
+/** Statements for the zk proof. */
+data class Attribute(
+    val key: String,
+    val value: ByteArray
+)
```

Comment:
We're generally trying to use `ByteString` instead of `ByteArray` so new code should by `ByteString`.. both here and possibly elsewhere... (also, it's generally not safe to use `ByteArray` in a data class)

Reply:
I made all the public class use ByteString, but kept ByteArray for some private classes since it is much easier to work with ByteArray in the native code.

Reply:
Sounds good to me.